fuel-indexer-database/sqlite/src/lib.rs

```diff
@@ -1,7 +1,11 @@
 use fuel_indexer_database_types::*;
 use fuel_indexer_lib::utils::sha256_digest;
 use sqlx::{pool::PoolConnection, types::JsonValue, Connection, Row, Sqlite, SqliteConnection};
-use tracing::info;
+use tokio::time::{sleep, Duration};
+use tracing::{info, warn};
+
+const RETRY_LIMIT: usize = 5;
+const INITIAL_RETRY_DELAY: u64 = 15;
```

Comment:
I would maybe lower this to something like `2`. If the DB doesn't connect in 30 seconds we have other issues. Not a blocker